miniirc_discord.py

```diff
@@ -90,14 +90,17 @@ def quote(self, *msg, force = None):
             if len(args) == 4:
                 EMBED_NOTICE = discord.Embed(title=args[0], description=args[1], colour=arg[2])
                 EMBED_NOTICE.set_author(name=args[3], icon_url=discord.client.user.default_avatar_url)
+                self._run(self._client.send_messge(chan, EMBED_NOTICE))
             else:
                 self.debug('Invalid call to EMBED_NOTICE')
         elif cmd in ('NAMES',):
-            discord.Member._dict_
-    
+            self._run(self._client.send_messge(chan, discord.Server.members))
```

Comment:
`NAMES` normally replies with `RPL_NAMREPLY` and `RPL_ENDOFNAMES` (see [the RFC](https://tools.ietf.org/html/rfc1459#section-4.2.5)).